### ISSUE TITLE
Where individual contributions come from feature updates

### ISSUE THREAD

**Issue by JonellaCulmer** [fecgov/fec-cms#3203]
**What we're after:**
As discussed in "[Decide on final acceptance criteria for Where contributions come from](https://github.com/fecgov/fec-cms/issues/3195)", [a list of four tasks](https://docs.google.com/document/d/19hJEhPYfSl__6Z9ukhk6WvBHVJZbqKns6r5rtMLHjEw/edit#heading=h.7ax27mky9im) still need to be completed prior to launch. After these four tasks are completed we will still have the feature under a feature flag in order to do any final testing that is required and necessary. 

### Related issues
- https://github.com/fecgov/fec-cms/issues/3195 Decide on final acceptance criteria...
- https://github.com/fecgov/fec-cms/issues/3146 Testing ticket

### Completion criteria

- [x] Open separate ticket for each of the identified acceptance criteria] (https://docs.google.com/document/d/19hJEhPYfSl__6Z9ukhk6WvBHVJZbqKns6r5rtMLHjEw/edit#) in MoSCoW document

### Future work

- [ ] Complete testing
- [ ] Open up followup tickets for [additional work](https://docs.google.com/document/d/19hJEhPYfSl__6Z9ukhk6WvBHVJZbqKns6r5rtMLHjEw/edit#) that "Should" be done.

**Comment by JonellaCulmer** [comment]
The last of the four tasks to complete this sprint is being handled here: https://github.com/fecgov/fec-cms/issues/3204

**Comment by JonellaCulmer** [comment]
All current/pending tickets needed to complete this work are here: 

Candidate corrections for Oracle data: https://github.com/fecgov/openFEC/issues/3986
Fix logic for browse individual contributions button on WCCF feature https://github.com/fecgov/fec-cms/issues/3204
Show specific coverage dates for each candidate https://github.com/fecgov/fec-cms/issues/3215
Update key to match colors displayed in map https://github.com/fecgov/fec-cms/issues/3216

**Comment by JonellaCulmer** [comment]
Closing in favor of the aforementioned tickets.
